libportability-gfx/src/lib.rs

```diff
@@ -269,101 +273,73 @@ pub const VK_MAX_EXTENSION_NAME_SIZE: ::std::os::raw::c_uint = 256;
 pub const VK_MAX_DESCRIPTION_SIZE: ::std::os::raw::c_uint = 256;
 pub const VK_KHR_surface: ::std::os::raw::c_uint = 1;
 pub const VK_KHR_SURFACE_SPEC_VERSION: ::std::os::raw::c_uint = 25;
-pub const VK_KHR_SURFACE_EXTENSION_NAME: &'static [u8; 15usize] =
-    b"VK_KHR_surface\x00";
+pub const VK_KHR_SURFACE_EXTENSION_NAME: &'static [u8; 15usize] = b"VK_KHR_surface\x00";
 pub const VK_KHR_WIN32_SURFACE_SPEC_VERSION: ::std::os::raw::c_uint = 6;
 pub const VK_MVK_MACOS_SURFACE_SPEC_VERSION: ::std::os::raw::c_uint = 2;
-pub const VK_KHR_WIN32_SURFACE_EXTENSION_NAME: &'static [u8; 21usize] =
-    b"VK_KHR_win32_surface\x00";
-pub const VK_MVK_MACOS_SURFACE_EXTENSION_NAME: &'static [u8; 21usize] =
-    b"VK_MVK_macos_surface\x00";
-pub const VK_EXT_METAL_SURFACE_EXTENSION_NAME: &'static [u8; 21usize] =
-    b"VK_EXT_metal_surface\x00";
+pub const VK_KHR_WIN32_SURFACE_EXTENSION_NAME: &'static [u8; 21usize] = b"VK_KHR_win32_surface\x00";
```

Comment:
eh, this is also, I think, looking better with the old code. Unfortunate, since I don't think if it's fixable.